website/docs/maintenance/backup_restore.md

```diff
@@ -10,7 +10,7 @@ If you are using `docker-compose`, you are using Docker Compose v1, which has be
 Create backup file `teslamate.bck`:
 
 ```bash
-docker compose exec -T database pg_dump -U teslamate teslamate > /backuplocation/teslamate.bck
+docker compose exec -T database pg_dumpall -U teslamate > /backuplocation/teslamate.bck
```

Comment:
This will work for upgrading to postgres 16, but not if someone comes from postgres 12 or older I beliefe.